src/file/numbering/numbering.ts

```diff
@@ -47,113 +47,115 @@ export class Numbering extends XmlComponent {
+                {
+                    level: 5,
+                    format: "bullet",
+                    text: "\u25A0",
+                    alignment: AlignmentType.LEFT,
+                    style: {
+                        paragraph: {
+                            indent: { left: 4320, hanging: 360 },
+                        },
                     },
                 },
-            },
-            {
-                level: 6,
-                format: "bullet",
-                text: "\u25CF",
-                alignment: AlignmentType.LEFT,
-                style: {
-                    paragraph: {
-                        indent: { left: 5040, hanging: 360 },
+                {
+                    level: 6,
+                    format: "bullet",
+                    text: "\u25CF",
+                    alignment: AlignmentType.LEFT,
+                    style: {
+                        paragraph: {
+                            indent: { left: 5040, hanging: 360 },
+                        },
                     },
                 },
-            },
-            {
-                level: 7,
-                format: "bullet",
-                text: "\u25CF",
-                alignment: AlignmentType.LEFT,
-                style: {
-                    paragraph: {
-                        indent: { left: 5760, hanging: 360 },
+                {
+                    level: 7,
+                    format: "bullet",
+                    text: "\u25CF",
+                    alignment: AlignmentType.LEFT,
+                    style: {
+                        paragraph: {
+                            indent: { left: 5760, hanging: 360 },
+                        },
                     },
                 },
-            },
-            {
-                level: 8,
-                format: "bullet",
-                text: "\u25CF",
-                alignment: AlignmentType.LEFT,
-                style: {
-                    paragraph: {
-                        indent: { left: 6480, hanging: 360 },
+                {
+                    level: 8,
+                    format: "bullet",
+                    text: "\u25CF",
+                    alignment: AlignmentType.LEFT,
+                    style: {
+                        paragraph: {
+                            indent: { left: 6480, hanging: 360 },
+                        },
                     },
                 },
-            },
-        ]);
+            ]);
 
-        this.createConcreteNumbering(abstractNumbering);
-
-        for (const con of options.config) {
-            const currentAbstractNumbering = this.createAbstractNumbering(con.levels);
-            this.createConcreteNumbering(currentAbstractNumbering, con.reference);
+            this.createConcreteNumbering(abstractNumbering);
+        } else {
+            for (const con of options.config) {
+                const currentAbstractNumbering = this.createAbstractNumbering(con.levels);
```

Comment:
can be inlined

Reply:
What can be inlined?

Reply:
never mind, was like this in the previous version.
`this.createAbstractNumbering(con.levels)` can be directly inlined into the following call instead of being assigned to a const. But that's a matter of style preference. I tend to avoid assigning variables (or consts as they usually also bear mutable state) because it's one less state to worry about. But here it really doesn't matter and we should probably stick to the style of the library.